### ISSUE TITLE
Running WordSpec tests from Intellij results in no tests detected, due to specifying erroneous test path

### ISSUE THREAD

**Issue by li-peng-dd** [kotest/kotest#5444]
I wrote up a dummy test project, using:

- io.kotest:kotest-runner-junit5-jvm:4.6.0
- kotlin 1.5.10
- kotest plugin 1.1.36-IC-2021.1
- Intellij IU-211.7442.40

I found that FunSpec works just fine, but running WordSpec just results in nothing ever being run, the test suite is just blank.

<img width="676" alt="Screen Shot 2021-06-01 at 9 31 34 PM" src="https://user-images.githubusercontent.com/53200271/120424886-172c0f80-c322-11eb-9ef1-249736e49c30.png">

I took a closer look at the test config:

<img width="727" alt="Screen Shot 2021-06-01 at 9 31 48 PM" src="https://user-images.githubusercontent.com/53200271/120424917-2743ef00-c322-11eb-8eb0-d477dff11aa9.png">

When I run the tests from gradle, I noticed that the test path is different:

```
MBP-0611 KoTestStuff % ./gradlew test

> Task :test FAILED

MyTest > hey there should > MyTest.do stuff FAILED
    org.opentest4j.AssertionFailedError at MyTest.kt:8
```

Note the test path `hey there should > MyTest.do stuff `

So I tried a custom test run with `hey there should` as the test path, instead of the `hey there` generated by the plugin. And it worked!

<img width="1310" alt="Screen Shot 2021-06-01 at 9 34 23 PM" src="https://user-images.githubusercontent.com/53200271/120425068-6d994e00-c322-11eb-9e2e-4ae817bb83db.png">

Not sure if KoTest  or the Plugin is  is generating the wrong test path here. 

The test is also broken if you run from the test level as well (in addition to being broken at the spec level), fixed it by using the test path `hey there should -- do stuff `. I think the "should" keyword is tripping something up here.

**Comment by li-peng-dd** [comment]
@sksamuel any ideas?

**Comment by sksamuel** [comment]
This is fixed in the latest plugin and kotest 5.0.0.

**Comment by li-peng-dd** [comment]
woo! thank you so much!
